src/hooks.rs

```diff
@@ -297,6 +605,84 @@ impl InnerConnection {
             }
         };
     }
+
+    pub fn authorizer<'c, F>(&'c mut self, authorizer: Option<F>)
+    where
+        F: for<'r> FnMut(AuthContext<'r>) -> Authorization + Send + RefUnwindSafe + 'static,
+    {
+        unsafe extern "C" fn call_boxed_closure<'c, F>(
+            p_arg: *mut c_void,
+            action_code: c_int,
+            action_arg1_str: *const c_char,
+            action_arg2_str: *const c_char,
+            db_str: *const c_char,
+            accessor_str: *const c_char,
```

Comment:
```
param1,
param2,
db_name,
trigger_name,
```

Reply:
I agree with these names, but they don't match the pre-existing style. For instance `db_str` came from `update_hook`, and `tbl_str` (from the same fn) suggests that the parameters are suffixed with their types. If you want the names changed here, I'd also like to go change the rest of them to keep the style consistent. What would you prefer?